src/main/java/org/apache/sling/feature/cpconverter/accesscontrol/DefaultAclManager.java

```diff
@@ -122,6 +134,8 @@ public void addRepoinitExtension(@NotNull List<VaultPackageAssembler> packageAss
                     .filter(entry -> getSystemUser(entry.getKey()).isPresent())
                     .map(Entry::getValue)
                     .flatMap(Collection::stream)
+                    // paths only should/need to be create with resource-based access control
+                    .filter(((Predicate<AccessControlEntry>) AccessControlEntry::isPrincipalBased).negate())
```

Comment:
if principal-based ac-setup is enforced for all service user permissions (and thus resource-based entries get converted to principal-based), no paths should be created either.... so in addition to filtering out principal-based entries, the whole block might be excluded if the enforce-prinicipal-based-ac-setup is turned on.

and we should have a test verifying this :)

Reply:
Yes, I think we can just wrap the complete block in an if statement.